README.md

```diff
@@ -53,7 +52,7 @@ in the Ansible community if you want to run the `devel` branch.
   to avoid duplicate efforts. This not only helps everyone
   know what is going on, but it also helps save time and effort if we decide
   some changes are needed.
-* For a list of email lists, IRC channels and Working Groups, see the
+* For a list of email lists, IRC channels, and Working Groups, see the
```

Comment:
```suggestion
* For a list of email lists, Matrix and IRC channels, and Working Groups, see the
```

Reply:
Right, `Matrix and IRC channels` is one list item.